ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -33,7 +33,7 @@ import "./views"
 StatusDialog {
     id: popup
 
-    property var preSelectedAccount: selectedAccount
+    property string preSelectedAccountAddress: store.selectedSenderAccountAddress
```

Comment:
I'm not sure about this.. I think we can have a corner case with a mistmach between `selectedAccount.address` and `preSelectedAccountAddress` and I think they should have the same value always. Meaning that:
- _preSelectedAccountAddress_ should be an alias for the _selectedAccount.address_
- Every time _selectedAccount_ changes, this _preSelectedAccountAddress_ should be updated
- Only in initialization of the popup, the _preSelectedAccountAddress_ should check the _store_ value

Reply:
There's only one purpose for all the `preSelected` properties and it's to configure the dialog at initialisation. But afterwards the user can choose any other account address, recipient and so on. And in this case the newly selected address is stored in the `selectedAccount`.

Maybe one refactoring idea would be to use the same properties for input and output.